chart/tests/test_rbac.py

```diff
@@ -0,0 +1,294 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import unittest
+
+import jmespath
+
+from tests.helm_template_generator import render_chart
+
+CLEANUP_DEPLOYMENT_KIND_NAME_TUPLES = [
```

Comment:
Curious about the name of this variable, why `CLEANUP_`?

Reply:
This was actually from @jaydesl's original PR where the test included the enabling of the cleanup job. Since it now also includes the `pgBouncer` deployment it admittedly doesn't carry the exact same weight as before. 

I could change it to `CLEANUP_PGBOUNCER_DEPLOYMENT_KIND_NAME_TUPLES`? Or maybe even `CLEANUP_PGBOUNCER_ENABLED_DEPLOYMENT_KIND_NAME_TUPLES`? Bit wordy but likely a bit more clear. Curious on your thoughts

Reply:
well it has lot more than that, it has statsd, webserver. etc.

maybe `DEPLOYMENT_NO_RBAC_NO_SA_KIND_NAME_TUPLES` to match the name of the test using it ?

Reply:
Only this comment is pending now :)